tests/terraform/modules/k3scluster/config/local.tfvars

```diff
@@ -0,0 +1,28 @@
+region = "us-east-2"
+qa_space = ""
+create_lb = false
+
+external_db_version = "5.7"
+instance_class = "db.t2.micro"
+db_group_name = "mysql5.7"
+engine_mode = "provisioned"
+db_username = ""
+db_password = ""
+
+username = ""
+password = ""
+
+ec2_instance_class = "t3a.medium"
```

Comment:
Can we move away from burstable instance types for tests? I think this is causing a lot of flakes when lack of credits causes the instances to get throttled for CPU or disk IO.

Reply:
The burstable types are due to cost primarily and have been OK for the most part thus far. For certain tests (sonobuoy) we should use dedicated CPU instance types. I think by default we should use these lower cost types and adjust as needed.